maquetado/index.html

```diff
@@ -8,6 +8,21 @@
  <link rel="stylesheet" type="text/css" href="scss/out/application.css">
 </head>
 <body>
-  <h1 class="title">Hello world</h1>      
+  <div>
+    <p class="text-big text-bold book-element">Título del libro</p>
+    <p class="text-grey book-element">(género)</p>
+  </div>
+  <div>
+    <p class="text-bold book-element">Autor del libro:</p>
+    <p class="text-grey book-element">Nombre del autor del libro</p>
+  </div>
+  <div>
+    <p class="text-bold book-element">Editorial:</p>
+    <p class="text-grey book-element">Nombre de la editorial</p>
+  </div>
+  <div>
+    <p class="text-bold book-element">Año de publicación:</p>
+    <p class="text-grey book-element">Año de publicación</p>
```

Comment:
You have the same text twice

Reply:
`Año de publicación: Año de publicación` is how it's displayed in the detail image.